public/stylesheets/main.css

```diff
@@ -12,4 +12,17 @@
 
 #delete-form {
     display: inline;
-}
\ No newline at end of file
+}
+
+.form-heading {
+    text-align: center;
+    margin-top: 30px;
+}
+
+.form-label {
+    width: 30%;
+    margin: 30px auto;
+}
+
```

Comment:
A single blank line at the end of the file is enough.

Reply:
Okay, I will leave it in the next commit.